.github/workflows/pypi-deploy.yml

```diff
@@ -9,17 +9,17 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      # checkout@v1 is required since versioneer uses git describe.
+      # checkout@v1 is used to that tags are more relaible.
```

Comment:
The checkout action has a history of messing up references to annotated tags which could affect scripts deriving versions from those tags but if the build system is just using any tag then it will probably work fine.  https://github.com/actions/checkout/issues/290

I think I can update the version here without issues, but you don't really know until you push a commit and it breaks.

Reply:
OK - I added the URL to the suggestion.